### DISCUSSION TITLE
Introductions!

### DISCUSSION THREAD

**Discussion by edakturk14** [BuidlGuidl/batch4.buidlguidl.com#1] [Announcements]
Hey everyone - It's Eda here, I've been part of the BuidlGuidl for almost 1.5 years! I'm a developer and I spend a lot of time [writing](https://mirror.xyz/edatweets.eth) about different topics and concepts in crypto. I enjoy working on developer education and creating content/documentation.

See you on Telegram!

**Comment by envilogger** [comment]
Hey all, my name is Anton and I'm in software development for almost 2 decades already. Mainly backend development in .net and java/scala and last decade in fintech field. 
Jumped into crypto dev a month ago and looking for interesting projects, but mainly learning yet

**Reply by carletex** [reply]
Welcome to the on-chain world!!

**Comment by BillSchumacher** [comment]
Hello, my name is Bill, I found the challenges about a month ago. I'm currently a Senior Software Engineer / Lead, I love to write code and build things,

I've been working on https://github.com/BillSchumacher/MintyBurny/ since completing the speedrun challenges and got a few contracts deployed.  It's completely lacking a frontend though.

Definitely had some challenges, I look forward to collaborating with yall.

**Reply by carletex** [reply]
Time spent building a full-stack dApp:

Create, debug, test & deploy a smart contract: 5%
Trying to center a div: 95 %

Welcome Bill!!

**Comment by ARJUN-R34** [comment]
Hello folks, my name is Arjun and I am a full stack (backend-focused) web3 engineer with over 5 years of experience.
I am the co-founder of [WalletX](https://getwalletx.com).
I love collaborating with fellow devs and building something meaningful.

Check out my website here - https://arjun.wtf

**Reply by carletex** [reply]
Welcome! Looking forward to learning from you!

**Comment by perstarkse** [comment]
Hi, looking forward to participating in this batch with you all. 

My name is Per and I’m a fullstack developer (primarily nodejs) since about a year ago. Been in crypto since 2017. Have previously worked as a clinical psychologist before transitioning to software development.

**Reply by carletex** [reply]
Love the transition!! :man_health_worker:

**Comment by carletex** [comment]
Hey all!! Thanks for the introductions!

My name is Carlos, I've been part of the BuidlGuidl since 2021. I've been a software developer for more than 10 years and went _onchain_ three years ago.

During this time I've built things like the [SpeedRunEthereum](https://speedrunethereum.com/) platform, the [BuidlGuidl App](https://app.buidlguidl.com/), [ABI Ninja](https://abi.ninja/) & [Scaffold-ETH 2](https://github.com/scaffold-eth/scaffold-eth-2). Inside the BuidlGuild, I also run the [Sand Garden](https://sandgarden.buidlguidl.com/).

I love learning and building things 👾

Alongside other BG members, I'll be helping you all to go through this batch. Hope you all have fun!

P.S. Just wrote this Twitter thread yesterday (my first thread, lol) about BG / Batches / Etc that might give you some insights: https://twitter.com/_carletex_/status/1777342905251729885

**Reply by pjvperes** [reply]
Nice to meet you, Carlos. I've heard about BuidlGuidl in ETHSamba, a brazilian hackathon at Rio de Janeiro, and we got 2°place in BuidlGuidl Track.

Now, I hope to be part of this community and actively help you with some contributions to this ecosystem to build amazing stuff!

**Comment by bhavyagor12** [comment]
Hello everyone!
I'm Bhavya Gor from Mumbai, India. I'm in my final year of studies and will be graduating this year. I've been a curious individual since childhood, fueled by my love for technology and computers, which began when I started playing Mario at the age of 4. Currently, I serve as a founding engineer at a web2 startup based in India. I have a keen interest in exploring blockchain technology and enjoy building things with it. My goal is to learn extensively in this field, make meaningful contributions, and have fun tackling exciting challenges along the way.

**Reply by carletex** [reply]
Welcome!! ;)

**Comment by yashgo0018** [comment]
Hi everyone,
My name is Yash. I am a blockchain & backend developer from last 5.5 years. For the last 1 year, I am going to a lot of hackathons and just travelling the world. I love travelling, meeting new people and making new friends. I am currently trying to learn zero knowledge proofs and deep learning. It is really nice to meet you all, would love to connect with you guys.
My linkedin: https://www.linkedin.com/in/yash-goyal-0018/
My twitter: https://twitter.com/yash_goyal_dev

**Reply by carletex** [reply]
Welcome! What backend stack are you familiar with?

**Reply by yashgo0018** [reply]
golang, node, python and learning rust

**Comment by ryanviana** [comment]
Hello everyone, 

I'm Ryan, nearing graduation at Computer Engineering in Brazil, with a background as a backend developer over the last two years. Recently, I've co-founded a software house with college colleagues and have spent the past year diving deep into blockchain through numerous hackathons, securing a few wins along the way. My experience has ignited a passion for blockchain technology, and I'm keen to contribute to the BuidlGuidl community. 

Looking forward to collaborating and exploring new frontiers in blockchain with you all. 

Connect with me on [LinkedIn](https://www.linkedin.com/in/ryanviana/).

**Reply by carletex** [reply]
Bem-vindo!! Hope you enjoy learning with the BG

**Reply by ryanviana** [reply]
Valeu! ;)

**Comment by vhforbes** [comment]
Hey guys! My name is Victor Hugo, I'm a self taught programmer who started in marketing and did a career transition to tech during the pandemic. I've been in tech for about 3 years and looking to make a transition to web3. I'm tired of big techs and looking to make a change and work in a environment more suited with my core beliefs.

Been following the Guild for about 2 years but just manage to have total focus now after a layoff! Some bad thing come to be good after all. Super passionate about the space and eager to learn.

Heres my [twitter(x)](https://twitter.com/vhforbes_)
and my [LinkedIn](https://www.linkedin.com/in/victor-hugo-forbes)

**Reply by carletex** [reply]
Bem-vindo Victor!

Glad you are back to BG 🤗

**Comment by Welith** [comment]
Hi all! I'm Boris, and I am a backend web2 engineer. I've been in the tech industry for over 5 years now, just picking up web3. Excited to be working on this project with all of you. I hope that we make some proper apps!

My twitter -> https://twitter.com/bo4ka7a

**Reply by carletex** [reply]
Welcome! Happy to have you here

**Comment by pjvperes** [comment]
Hi everyone. I'm Pedro from Brazil.

I've heard about BuidlGuidl in ETHSamba, a brazilian hackathon at Rio de Janeiro, and we got 2°place in BuidlGuidl Track. So, I got excited to contribute to this ecosystem and join this community.

I already worked as blockchain developer and as investment analyst at a crypto Venture Capital. Recently, I left my job to found a software house with @ryanviana and @gugasanchez

Pleasure to meet you, guys. Feel free to send me a message on Telegram if you have any doubts (tg: @pjvperes)

Twitter (X): https://twitter.com/pedroncperes

**Comment by gugasanchez** [comment]
Pleasure everyone!

I'm Gustavo, a developer and Computer Engineering student from University of Sao Paulo, Brazil! As my friend @pjvperes said in his comment, we're cofounding a software house alongside @ryanviana which is also in this batch.

We've been developing on top of BuidlGuidl's Scaffold Eth 2 for quite some months now, which helped us a lot achieve amazing results in hackathons and our company's projects.

Very excited to be part of this batch and contribute to our ecosystem! And a huge thanks to Juan David, a long time BuidlGuidl contributor that introduced me to this journey!

LinkedIn: https://www.linkedin.com/in/gustavo-barbosa-sanchez/

**Comment by bhargavkakadiya** [comment]
gm
my name is Bhargav Kakadiya (BK)
I live in Vancouver and work as a full stack engineer in SaaS company focusing on building science
previously I have worked on data and tokenomics consulting and currently I am focusing on improving my web3 skills especially writing smart contracts and understanding the core challenges around scalability.  
X: https://x.com/bgkakadiya

**Comment by mrhouzlane** [comment]
hi frens, 
I'm mehdi a blockchain & backend dev focused on learning solidity & security and i'm building stormbit a p2p lending protocol on scaffold.

 - Project : [Stormbit](https://github.com/Quantum3-Labs/Stormbit-monorepo)
 - Socials : [X](https://x.com/Mehdi96_)

**Comment by gilbertsahumada** [comment]
Hi everyone,

I'm Gilberts Ahumada from Chile. I've been developing software for +8 years and it's been 3 years in blockchain. Currently I'm running my own business and also creating content about blockchain dev in spanish. I've been following the buidlguidl and scaffold-eth for a while but after eth denver I decided to get more into it and here I'm.  

nice to meet you all , here are my socials [youtube](https://www.youtube.com/@observdev) and [X](https://twitter.com/teterabOb)

**Comment by JulioMCruz** [comment]
Hello All,
My name is Julio Cruz and I am from Virginia, USA. I originally hail from Peru and I am delighted to be here to collaborate with you all. I have been working as a software engineer for over 20 years now, specifically in Cloud solutions for both government and private organizations. Additionally, I have around 2.5 years of experience in Onchain development. Please do not hesitate to send me a direct message if you would like to connect with me on [X](https://twitter.com/JulioMCruz) or [Farcaster](https://warpcast.com/juliomcruz)
